data/json/items/armor.json

```diff
@@ -12247,14 +12247,14 @@
         "material" : ["cotton", "leather"],
         "volume" : 10,
         "cutting" : 0,
-        "warmth" : 35,
+        "warmth" : 40,
         "phase" : "solid",
         "environmental_protection" : 4,
-        "encumbrance" : 3,
+        "encumbrance" : 2,
         "bashing" : 0,
         "flags" : ["VARSIZE", "WATERPROOF"],
         "coverage" : 95,
-        "material_thickness" : 4
+        "material_thickness" : 5
```

Comment:
It makes sense to protect both your head and body when making a specialized suit, even it it's fitted for scavenging and not combat.

I'm not sure I like the nomad gear this way at all, but I suppose I'll revert all the other stat changes (except for lower encumbrance).